helm/defaultClusterIssuer/templates/clusterIssuerHTTP.yaml

```diff
@@ -0,0 +1,26 @@
+{{- $outer := . -}}
+{{- range $clusterIssuerName, $clusterIssuerValues := .Values.http }}
+{{- with (merge (dict "clusterIssuerName" $clusterIssuerName "clusterIssuerValues" $clusterIssuerValues) $outer) }}
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: {{ .clusterIssuerName }}
+  labels:
+    app: {{ include "defaultClusterIssuer.name" . }}
+    app.kubernetes.io/name: {{ include "defaultClusterIssuer.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "clusterIssuer"
+    helm.sh/chart: {{ include "defaultClusterIssuer.chart" . }}
+spec:
+  acme:
+    {{- toYaml .clusterIssuerValues.acme | nindent 4 }}
+    solvers:
+      - http01:
+          ingress:
+            {{- if .clusterIssuerValues.ingressClassName }}
+            ingressClassName: {{ .clusterIssuerValues.ingressClassName }}
+            {{- end }}
```

Comment:
Please add more configuration option for HTTP solver:
- serviceType
- podTemplate
- ingressTemplate
- class
- name

By default, we should use chart labels in podTemplate and ingressTemplate with an option that allow users to specify their own labels as well:
```
  labels:
    app: {{ include "defaultClusterIssuer.name" . }}
    app.kubernetes.io/name: {{ include "defaultClusterIssuer.name" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
    app.kubernetes.io/component: "clusterIssuer"
    helm.sh/chart: {{ include "defaultClusterIssuer.chart" . }}
```